tutorials/http/http-server/Makefile

```diff
@@ -3,7 +3,7 @@ DELETE = rm -rf                   # Command to remove files
 OUT ?= -o $(PROG)                 # Compiler argument for output file
 SOURCES = main.c mongoose.c       # Source code files
 CFLAGS = -W -Wall -Wextra -g -I.  # Build options
-CFLAGS += -fsanitize=address,undefined,alignment
+#CFLAGS += -fsanitize=address,undefined,alignment
```

Comment:
yesterday added, today removed. Why removing it ? Windoze ?

Reply:
Yeah, windows build.
Added for debugging